FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -13,87 +9,132 @@ class ProgramRun
     {
         public void Run()
         {
-            Vector vectorFirst = EnterUserData("first");
-            Vector vectorSecond = EnterUserData("second");
+            Vector vectorFirst = GetUserVector("first");
+            Vector vectorSecond = GetUserVector("second");
+
+            ShowVectors(vectorFirst, vectorSecond);
```

Comment:
So many methods that shows the results of different operations. I think it's better to export them into another method. This method will contain the rules for displaying: which operations, which order etc.